angular-spa-ui/js/admin-app/content-manager-module/contentManager-ctrl.js

```diff
@@ -1,90 +1,104 @@
 "use strict";
 
-module.exports = function ($scope, $state, promises, $stateParams, cmService, configService, contentStorage) {
+module.exports = function ($scope, $state, promises, $stateParams, cmService, configService, contentStorage, rlService) {
+
+    var vm = this;
+
+    vm.viewApi = {
+        isActive: function (tabs) {
+            // Set active tab
+            return cmService.isActiveTab(tabs, contentStorage.params.searchIn);
+        },
+        changeTab: function (place) {
+            // Tab click go to state
+            if (contentStorage.params.searchIn == place)
+                return false;
+
+            $state.go(
+                'content', {
+                    searchIn: place
+                }, {
+                    reload: true,
+                    inherit: false
+                }
+            );
+
+        }
+    }
+
+    vm.model = {
```

Comment:
That is not a model. The MVC Model contains all application logic (business logic, validation logic, and data access logic), except pure view and controller logic.